Libraries/Components/DatePicker/DatePickerIOS.ios.js

```diff
@@ -21,10 +21,15 @@ const View = require('View');
 const requireNativeComponent = require('requireNativeComponent');
 
 import type {ViewProps} from 'ViewPropTypes';
+import type {SyntheticEvent} from 'CoreEventTypes';
 
 const RCTDatePickerIOS = requireNativeComponent('RCTDatePicker');
 
-type Event = Object;
+type Event = SyntheticEvent<
+  $ReadOnly<{|
+    timestamp: string | number,
```

Comment:
Could you link to the file(s) that shows that this is the type for `timestamp` from native?

Reply:
It looks like `timestamp` can only be a number:

https://github.com/facebook/react-native/blob/1151c096dab17e5d9a6ac05b61aacecd4305f3db/React/Views/RCTDatePicker.m#L32-L37

Reply:
OK! I fixed.